### ISSUE TITLE
A new error code for model import failures due to encrypted file

### ISSUE THREAD

**Issue by carmenfan** [3drepo/3drepobouncer#667]
### Description
Occasionally we get files (we've seen it on nwds, dwgs) that are encrypted and thus cannot import:
![image](https://github.com/3drepo/3drepobouncer/assets/11945337/6f7ce65d-716b-444e-bbdd-4dd7aa43eba3)

These are currently returning error code 5, but we should make a new error code to show it's a problem user can address.

### Goals
- [ ] As a user I would like to be informed if the cause of an import failure is because the file is encrypted.

### Tasks
TBC

**Comment by sebjf** [comment]
I have made the changes for NWDs and added a unit test.

For DWGs, support for password protection was [removed in 2016](https://www.autodesk.com/support/technical/article/caas/sfdcarticles/sfdcarticles/Password-protection-feature-in-AutoCAD.html) and Autodesk will not provide a version earlier than 2017.

I tried [this application](https://www.autodwg.com/dwglock/) which did successfully protect a file. However, while ODA can open it  if password protection is not enabled, when it is enabled ODA fails with the code `fail`, and not `bad_password` or `encrypted_file`.

**Comment by sebjf** [comment]
I also exported a password protected DWG from [this sample](https://docs.opendesign.com/td/td_sample_OdWriteEx.html) but we get the same response.

It seems the `OdDgnExport` class is not reporting the error correctly.

**Comment by sebjf** [comment]
Detection of password protected DWGs and the new error code have been added in #679. That branch has been merged into this one and encrypted NWD detection added. #679 must be merged before this branch.

**Comment by carmenfan** [comment]
merged into #679
